website/docs/cli/config/config-file.mdx

```diff
@@ -115,12 +115,22 @@ is available at multiple hostnames, use only one of them consistently.
 Terraform Cloud responds to API calls at both its current hostname
 `app.terraform.io`, and its historical hostname `atlas.hashicorp.com`.
 
-### Credentials Helpers
+### Environment Variable Credentials
 
 If you would prefer not to store your API tokens directly in the CLI
-configuration as described in the previous section, you can optionally instruct
-Terraform to use a different credentials storage mechanism by configuring a
-special kind of plugin program called a _credentials helper_.
+configuration, you may use a host-specific environment variable. For example,
+the value of a variable named `TF_TOKEN_app_terraform_io` will be used as an
+Authorization: Bearer token whenever making service requests to the hostname
+"app.terraform.io".
+
+Internationalized domain names should be converted to their [punycode equivalent](https://www.charset.org/punycode)
+with an ACE prefix when used as a variable name. For example, token credentials
+for 例えば.com should be set in a variable called `TF_TOKEN_xn--r8j3dr99h_com`.
```

Comment:
```suggestion
Convert domain names to their [punycode equivalent](https://www.charset.org/punycode) and include an ACE prefix when using them as a variable name. For example, set the token credentials for 例えば.com in a variable called `TF_TOKEN_xn--r8j3dr99h_com`.
```

Reply:
Thanks for your suggestions! I think something was lost in this modification, because this only applies to domain names containing non-ASCII characters, typical of domain names with accent marks or unicode characters for non-latin character sets. Should it say...

> Internationalized domain names containing non-ASCII chacters should be converted

?

Reply:
Is "internationalized" a common term? If not, I think "Convert domain names with non-ASCII characters to...."  

Thank you for pushing back on that!

Reply:
"[Internationalized domain name](https://en.wikipedia.org/wiki/Internationalized_domain_name)" as a whole is the particular jargon phrase used to refer to the family of specifications that this "punycode" formatting is a part of. Since we're already linking to a reference for "punycode" anyway we may not need to _also_ say "Internationalized domain name", but if we do then I'd suggest linking to a definition for it so it's clearer it's not just a regular adjective modifying "domain name".